app/workers/receive_base.rb

```diff
@@ -26,6 +26,7 @@ def filter_errors_for_retry
            DiasporaFederation::Salmon::InvalidEncoding,
            Diaspora::Federation::AuthorIgnored,
            Diaspora::Federation::InvalidAuthor,
+            Diaspora::Federation::PodBlocked,
```

Comment:
Layout/ArrayAlignment: Align the elements of an array literal if they span more than one line.